concent_api/core/message_handlers.py

```diff
@@ -765,7 +765,7 @@ def handle_send_force_payment(
 
     if not (
         are_keys_and_addresses_unique_in_message_subtask_results_accepted(client_message.subtask_results_accepted_list) and
-        are_subtask_results_accepted_messages_signed_by_the_same_requestor(client_message.subtask_results_accepted_list)
+        are_subtask_results_accepted_messages_properly_signed(client_message.subtask_results_accepted_list)
```

Comment:
This function name would be more describing if called
```
are_subtask_results_accepted_messages_signed_by_the_same_requestor_or_concent
```

Reply:
ok, I will change it.

Reply:
What about `_or_concent` part?